File: JavaParser/Print_Statement.java

public class Print_Statement implements Statement
{
	private Expression expr1;
	
	public Print_Statement()
	{	}
	
	public Print_Statement(Expression expr1)
	{
		if (expr1 == null)
			throw new IllegalArgumentException("expr1 error /PrintStatement");
		
		this.expr1 = expr1;
	}
	
	@Override
	public void execute()
	{
		//print a statement
		System.out.println(expr1.evaluate());
	}
}

File: PythonParser/PrintStatement.py
__author__ = 'ravipatel'

from Statement import Statement

class PrintStatement(Statement):

    def __init__(self, expr1):
        if (expr1 is None):
            raise Exception("Expr1 is wrong in print statement")
        self.expr1 = expr1

    def execute(self):
        print(self.expr1.evaluate())